pyspectral/utils.py

```diff
@@ -554,3 +551,24 @@ def get_logger(name):
     if not log.handlers:
         log.addHandler(NullHandler())
     return log
+
+
+def get_wave_range(in_chan, threshold=0.15):
+    """Return central, min and max wavelength in an RSR greater than threshold.
+
+    An RSR function will generally start near zero, increase to a maximum and
+    then drop back to near zero. This function takes advantage of this to find
+    the first and last points where the RSR is greater than a threshold. These
+    points are then defined as the minimum and maximum wavelengths for a
+    given channel, and can be used, for example, in Satpy reader YAML files.
```

Comment:
```suggestion
    given channel, and can be used, for example, in Satpy reader YAML files.
    
```
This should fix the Codacy complaint.